src/iuse.cpp

```diff
@@ -433,6 +433,8 @@ int iuse::ecig( player *p, item *it, bool, const tripoint & )
             p->add_msg_if_player( m_neutral,
                                   _( "You inhale some vapor from your advanced electronic cigarette." ) );
             p->use_charges( "nicotine_liquid", 1 );
+            item dummy_ecig = item("ecig", int(calendar::turn));
```

Comment:
```suggestion
            item dummy_ecig = item( "ecig", calendar::turn );
```

Reply:
Cast to int is unnecessary - item constructor takes time_point.

Reply:
In my defense, with this one I literally copied that item creation line from the section immediately above.

Reply:
Noone's attacking you :)

Existing code can (and will) be bad.